src/content/platform-integration/web/wasm.md

```diff
@@ -64,12 +73,12 @@ the `beta` channel (preferred) or `main`.
 
 To learn more about Flutter build release channels and how to switch to
 the `beta` channel, check out the
-[Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels).
+[Flutter wiki]({{site.github}}/flutter/flutter/wiki/Flutter-build-release-channels).
```

Comment:
**question:** Here and elsewhere, you point to a Wiki article that's almost 6 years old. Some of the information is [covered in the docs now](https://docs.flutter.dev/testing/build-modes) which is also 5+ years old. Should we link to the docs page first and look at updating it and bring some of the Wiki content into docs?

Reply:
Yes this has always been an awkward discrepency. We should update the docs with any relevant information. I'll defer that to separate, follow-up work though as those links are across the website.